recipes/server.rb

```diff
@@ -29,14 +31,14 @@
     recursive true
     mode '0700'
     owner 'root'
-    group 'root'
+    group 0
```

Comment:
Instead of using group 0 you can use the node['root_group'] attribute that returns the appropriate group per platform

Example on my mac:

chef > node['root_group']
 => "wheel"